src/CommunityToolkit.Maui/Alerts/Snackbar/Snackbar.windows.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Toolkit.Uwp.Notifications;
+using Windows.UI.Notifications;
+
+namespace CommunityToolkit.Maui.Alerts.Snackbar;
+
+public partial class Snackbar
```

Comment:
Can you ensure this code works on Windows?

When I run this in the Sample App on Windows, I don't get a toast or a snackbar. Nothing happens.

![ScreenFlow](https://user-images.githubusercontent.com/13558917/143681524-1ac46990-59a0-454d-8162-21c742d631d1.gif)

Reply:
in Windows, Toast is like a notification. Please disable silent mode

Reply:
Ah - user error! 🤦‍♂️ 

I wish I would've asked you sooner before spending an hour debugging it 😆

Reply:
After disabling silent mode, I get a weird behavior:
When I click `OK`, a new instance of the app appears. 

Do you know why this is happening? 

![ScreenFlow](https://user-images.githubusercontent.com/13558917/143682215-7adbb1fd-4875-4f73-8cd2-15857dc4d848.gif)

Reply:
replied in Discord, but it works like this: https://docs.microsoft.com/en-us/windows/apps/design/shell/tiles-and-notifications/send-local-toast?tabs=desktop#step-3-handling-activation

Reply:
Got it.

This behavior on Windows is different from iOS, MacCatalyst and Android then. I'm not a Windows dev or a heavy Windows user, so I'm not sure what Windows developers/users would expect. 

This'll be important to note in the documentation.

Could you also update the text on the `SnackbarPage` to inform Windows users to disable silent mode so dummies like me don't think it's broken 😃